src/odemis/driver/test/semcomedi_test.py

```diff
@@ -205,6 +209,21 @@ def test_ttl(self):
             self.scanner.external.value = v
             self.scanner.blanker.value = v
 
+    def test_magnification(self):
+        pxs_orig = self.scanner.pixelSize.value
+        mag_orig = self.scanner.magnification.value
+
+        self.assertEqual(pxs_orig[0], pxs_orig[1])
+
+        # Mag x 2 => pixel size / 2
+        self.scanner.magnification.value *= 2
+        self.assertAlmostEqual(mag_orig * 2, self.scanner.magnification.value)
+        new_pxs = self.scanner.pixelSize.value
+        self.assertAlmostEqual(pxs_orig[1] / 2, new_pxs[0])
+        self.assertEqual(pxs_orig[0], pxs_orig[1])
```

Comment:
Why don´t you do self.assertEqual(new_pxs[0], new_pxs[1]) ?

Reply:
that's a copy-paste mistake. Now fixed.